src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -22,10 +24,28 @@ export type Props = PlainButtonProps & {
 
 export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
 
-export function SidebarNavSign({ intl, status, targetingApi, ...rest }: Props) {
+export function SidebarNavSign({ blockedReason, intl, isDisabled, status, targetingApi, ...rest }: Props) {
     const isTargeted = targetingApi && targetingApi.canShow;
-    const FtuxTooltip = isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
     const label = intl.formatMessage(status === 'active' ? messages.boxSignSignature : messages.boxSignRequest);
+    let tooltipMessage = label;
+
+    let isSignDisabled = false;
+    switch (blockedReason) {
+        case 'shield-download':
+        case 'shared-link':
+            isSignDisabled = true;
```

Comment:
Seems like `const isSignDisabled = !!blockedReason;` up top could work to avoid these lines.

Reply:
i'm passing in 'none' right now, but that can be changed in EUA

Reply:
Sounds good. Let's update EUA to pass in a falsy value to simplify things, in that case.